test/kaldi_compatibility_impl.py

```diff
@@ -48,6 +89,8 @@ def _run_kaldi(command, input_type, input_value):
 
 
 class Kaldi(common_utils.TestBaseMixin):
+    test_8000_filepath = common_utils.get_asset_path('kaldi_file_8000.wav')
```

Comment:
Please keep this inside of test each test. The intended usage of asset file is not immediately clear when someone else work on adding a new test, so it is not very suited to be common property.